packages/firestore/test/integration/api/database.test.ts

```diff
@@ -20,14 +20,19 @@ import * as firestore from 'firestore';
 import { Deferred } from '../../../src/util/promise';
 import { asyncIt } from '../../util/helpers';
 import firebase from '../util/firebase_export';
-import { apiDescribe, withTestCollection, withTestDb } from '../util/helpers';
+import {
+  apiDescribe,
+  withTestCollection,
+  withTestDb,
+  withTestDoc
+} from '../util/helpers';
 import { Firestore } from '../../../src/api/database';
 
 apiDescribe('Database', persistence => {
   asyncIt('can set a document', () => {
-    return withTestDb(persistence, db => {
-      return db.doc('rooms/Eros').set({
-        desc: 'Stuff related to Eros project...',
+    return withTestDoc(persistence, docRef => {
```

Comment:
Probably not worth the effort, but it would be niceish if we picked either docRef or doc and not both :)

Reply:
Yeah.  If we did that I'd say we should do it in a separate PR...  but I don't think it's worth the effort. :-)